extensions/VisualEditor/modules/ve-mw/init/ve.init.mw.Target.js

```diff
@@ -1402,6 +1406,59 @@ ve.init.mw.Target.prototype.tryWithPreparedCacheKey = function ( doc, options, e
 };
 
 /**
+ * Send an AJAX request to the MediaWiki API.
+ *
+ * This method has special behavior for certain options. If the request type is POST, then
+ * contentType will default to multipart/form-data. If the content type is multipart/form-data,
+ * then the necessary emulation will be performed to make this content type actually work.
+ *
+ * @param {Object} data Query string parameters (for GET requests) or POST data (for POST requests)
+ * @param {Object} [settings] Additional AJAX settings, or overrides of default settings
+ * @return {jqXHR} Return value of $.ajax()
+ */
+ve.init.mw.Target.static.apiRequest = function ( data, settings ) {
```

Comment:
Later we can move it to our target just to minimize amount of change to the core